Octokit/Models/Request/IssueRequest.cs

```diff
@@ -121,23 +121,45 @@ public enum IssueFilter
     }
 
     /// <summary>
-    /// The range of states that an issue can be in.
+    /// Search API Filter
```

Comment:
This comment seems to be incorrect - it's used for issues/milestones/pull requests.